purchases/src/main/kotlin/com/revenuecat/purchases/PurchasesOrchestrator.kt

```diff
@@ -1154,6 +1186,8 @@ internal class PurchasesOrchestrator constructor(
 
         const val frameworkVersion = Config.frameworkVersion
 
+        const val lastSyncAttributesAndOfferingsTimeout = 60000
```

Comment:
Nitpick, but we could use the `Duration` types for these comparisons, so this could for example be `60.seconds`, and then in the other places we could also convert to `Duration`, to be more clear on the type.